tutorials/cyber/network_mapping/lanl_network_mapping_using_rapids.ipynb

```diff
@@ -0,0 +1,781 @@
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cg_edges_gdf['src_idx'] = cg_edges_gdf['src_idx'].astype('int32')\n",
+    "cg_edges_gdf['dst_idx'] = cg_edges_gdf['dst_idx'].astype('int32')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run cuGraph PageRank\n",
+    "\n",
+    "Next we create our graph and run pagerank."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "G = cugraph.Graph()\n",
+    "G.add_edge_list(cg_edges_gdf['src_idx'], cg_edges_gdf['dst_idx'], None)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 104 ms, sys: 8 ms, total: 112 ms\n",
+      "Wall time: 109 ms\n"
+     ]
+    }
+   ],
+   "source": [
+    "%time pr_gdf = cugraph.pagerank(G, alpha=0.85, max_iter=500, tol=1.0e-05)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "    vertex       pagerank\n",
+      "0       0  0.00021846234\n",
+      "1       1  0.00021846234\n",
+      "2       2  0.00021846234\n",
+      "3       3  0.00021846234\n",
+      "4       4  0.00021846234\n",
+      "5       5  0.00021846234\n",
+      "6       6  0.00021846234\n",
+      "7       7  0.00021846234\n",
+      "8       8  0.00021846234\n",
+      "9       9  0.00021846234\n",
+      "[2760 more rows]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(pr_gdf)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Add PageRank scores to node list"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pr_gdf['idx'] = pr_gdf['vertex'].astype('int64')\n",
+    "all_nodes_gdf = all_nodes_gdf.merge(pr_gdf, on=['idx'], how='left')\n",
+    "all_nodes_gdf = all_nodes_gdf.drop(['vertex'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Graphistry Viz\n",
+    "\n",
+    "We use [Graphistry](https://www.graphistry.com/) to visualize the network mapping graph. A snapshot of the graph constructed from this notebook can be viewed below. Run the code below to contruct a graph using Graphistry.\n",
+    "\n",
+    "  ![Network Mapping Graphistry Image 1](images/graphistry1.png \"Network Mapping using Graphistry\")    \n",
+    "  \n",
+    "  \n",
+    "Zoom in to search for interesting subgraphs.\n",
+    "\n",
+    "![Network Mapping Graphistry Image 2](images/graphistry2.png \"Network Mapping using Graphistry\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import graphistry"
```

Comment:
Can you update the Dockerfile to add this package to the conda environment, or include comments/a link for where the user can get this package?

Reply:
Will do, thanks